src/main/java/com/atlassian/bitbucket/jenkins/internal/model/BitbucketSingleTag.java

```diff
@@ -0,0 +1,58 @@
+package com.atlassian.bitbucket.jenkins.internal.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class BitbucketSingleTag {
```

Comment:
I'm a bit confused, do we need this? It looks like this is the output of a page of tags, but we know the the page will contain either 0 or 1 values, so I wonder if we can just pull the element out of the page in the client?

Also I should note that this _isn't_ a `BitbucketTag`, it's the `BitbucketCommit` associated with a tag- the fields are the same so I think if we need this array representation, we're better of returning a `List<BitbucketCommit>`- that's also a bit clearer to the consumer.